pkgs/top-level/python-packages.nix

```
@@ -1749,6 +1749,8 @@ in {
 
   cozy = callPackage ../development/python-modules/cozy { };
 
+  cppe = toPythonModule (pkgs.cppe.override { enablePython = true; });
```

The python version also needs to be set, or else you will get `pkgs.python3` for all interpreter versions
```suggestion
  cppe = toPythonModule (pkgs.cppe.override {
    enablePython = true;
    python3 = python;
  });
```

---

I've passed python explicitly, however the PySCF `pythonCatchConflictsPhase` now complains about duplicate dependencies in the closure (pybind, numpy, scipy, h5py and six), coming in directly as raw python packages, and wrapped into a python env (I think from `python.withPackages`). I'm now passing all Python packages directly, as nghttp2 does it [here](https://github.com/NixOS/nixpkgs/blob/b67e752c29f18a0ca5534a07661366d6a2c2e649/pkgs/development/libraries/nghttp2/default.nix#L12), and this reduces the conflicts to `pybind11`. But still there is the duplicate and I am not sure where this comes from. Any idea?